### PR TITLE
#160577594 Enable users to see their article stats

### PR DESCRIPTION
**Pull request by mutaimwiti**
**What does this PR do?**

This PR adds functionality that allows users to see how their articles are performing on the platform on a single dashboard.

**Description of Task to be completed?**

The article stats data gives the user data about the performance of their articles. Only the owner is able to see this data. This means that one has to be logged in to view their reading stats.

**How should this be manually tested?**

This feature can be manually tested by opening this PR's [review app](https://ah-cd-frontend-staging-pr-35.herokuapp.com/).
1. Log in to the app.
2. Click on the `Stats` link on the NavBar. You should be taken to a page that displays your article stats.

**Any background context you want to provide?**

When a backend PR adding like count, dislike count and rating stats is merged, they will also be added.

**What are the relevant pivotal tracker stories?**

[#160577594](https://www.pivotaltracker.com/story/show/160577594)

**Screenshots**
NavBar link
screen shot 2018-11-27 at 23 33 15

Loading stats
screen shot 2018-11-23 at 11 56 31

No stats
screen shot 2018-11-23 at 08 38 53

Stats on display
screen shot 2018-11-23 at 11 53 09

Refreshing stats - notice the preloader on the NavBar
screen shot 2018-11-27 at 23 35 56

### CODE REVIEW

**Review by C3real-kill3r — APPROVED** [review]

**Review by YAHYA-H — APPROVED** [review]

**Review by gitaumoses4 — CHANGES_REQUESTED** [review]
Good work on the task so far. 
*Stats Refreshing Recommendation*
I would advice when the data is refreshing you could use the navigation bar loader which would be more user friendly since it allows the user to continue exploring the page as the new data is being loaded.

**Comment by gitaumoses4** [comment]
Hello @mutaimwiti did you implement the refreshing?

**Comment by mutaimwiti** [comment]
> Hello @mutaimwiti did you implement the refreshing?

Not yet @gitaumoses4 I will be doing that today.

**Comment by mutaimwiti** [comment]
> Hello @mutaimwiti did you implement the refreshing?

It is now done @gitaumoses4

File: src/containers/Stats/state/actions.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  return api.get('/article-stats/')
+    .then(({ data }) => {
+      dispatch(fetchStatsSuccess(data));
+    })
+    .catch(() => {
+      dispatch(fetchStatsFailure());
+    });
+};
+
+// refresh stats
```
Comment (paulrimiru) [inline src/containers/Stats/state/actions.js:36]: REmove inline comment

**Review by paulrimiru — COMMENTED** [review]
Good work

**Review by gitaumoses4 — APPROVED** [review]

**Pull request merged**